src/main/resources/io/jenkins/plugins/analysis/core/steps/IssuesRecorder/config.jelly

```diff
@@ -17,6 +17,11 @@
     <f:checkbox />
   </f:entry>
 
+   <f:entry title="${%title.failBuildOnError}" description="${%description.failBuildOnError}" field="failBuildOnError">
+      <f:checkbox />
+    </f:entry>
+
+
```

Comment:
Sorry, this is my fault: we should place this entry into the file [publish-parameters.jelly](https://github.com/jenkinsci/warnings-ng-plugin/blob/master/src/main/resources/issues/publish-parameters.jelly) (at line 7). Then it will be used automatically by the `IssuesRecorder`, `PublishIssuesStep`, `RecordIssuesStep`.

Reply:
Resolved :+1: